scenarios/pravega-flink-connector-sql-samples/src/main/java/io/pravega/connectors/nytaxi/PopularDestinationQuery.java

```diff
@@ -36,29 +30,12 @@ public PopularDestinationQuery (String ... args) {
     @Override
     public void handleRequest() {
 
-        Schema schema = TripRecord.getSchemaWithPickupTimeAsRowTime();
-
         StreamExecutionEnvironment env = getStreamExecutionEnvironment();
 
         // create a TableEnvironment
-        StreamTableEnvironment tEnv = StreamTableEnvironment.create(
-                env,
-                EnvironmentSettings.newInstance()
-                        .useBlinkPlanner()
-                        .inStreamingMode()
-                        .build()
-        );
-
-        Pravega pravega = new Pravega();
```

Comment:
If we remove this part that configures scope/stream, then where are we going to be specifying it for the given query?

Reply:
The part I removed here is just the table connection establishment, and now we use the new Table API here, so it can be simplified to Flink SQL `CREATE TABLE` DDL. This also helps that I can abstract the same connection for all the queries in the same `AbstractHandler` class.
Scope and stream specifications are done here: https://github.com/pravega/pravega-samples/pull/275/files/5e22f5f58fdb3e7afea8919eb5ccb958d7c09f27#diff-801b8c5bcee3c1d0757b4e2bf591e7cad734b92a3f18a8c22607bd8a77635f2bR105
From the logic in `AbstractHandler` class, they are still remained configurable as Flink parameters just as-is, so the usage is not changed.